modules/core/src/lib/constants.ts

```diff
@@ -101,12 +108,27 @@ export const UNIT = {
 } as const;
 
 export const EVENTS = {
-  click: {handler: 'onClick'},
+  tap: {handler: 'onClick'},
```

Comment:
This is a very common event. Could changing it break apps?

Reply:
In the previous version, mjolnir defined preset recognizers and some "aliases", including tap <-> click, doubletap <-> dblclick

This is no longer the case, so I guess we want to pick one that breaks less apps?

`eventManager` is private and apps are not supposed to attach their own event listener. A custom Controller class has the option to listen to additional events, so in theory some will see the breaking change, but I don't imagine tap/click is utilized by many.